package.json

```diff
@@ -51,7 +51,10 @@
         "never"
       ],
       "import/prefer-default-export": "off",
-      "no-param-reassign": "off"
+      "no-param-reassign": "off",
```

Comment:
these are removed because we use prettier

Reply:
@fabienjuif It was quite annoying to repeat a directive on every field, this is exactly what I was looking for, I spent 3 days in vain trying to achieve that. Thank you so much.

Could you please publish it.

Reply:
I was testing it, it works like a charm, and thanks the code is more readable now

Reply:
Thank you ❤️ 

I'll publish it ASAP